code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -726,7 +726,7 @@
 	affected_mob.adjust_temp_blindness(-4 SECONDS * REM * seconds_per_tick * normalized_purity)
 	affected_mob.adjust_eye_blur(-4 SECONDS * REM * seconds_per_tick * normalized_purity)
 	var/obj/item/organ/internal/eyes/eyes = affected_mob.get_organ_slot(ORGAN_SLOT_EYES)
-	if(eyes)
+	if(eyes && !IS_ROBOTIC_ORGAN(eyes))
```

Comment:
Likewise this one should be moved down to the `if(eyes.damaged ...)` check instead, since the healing part already checks `affected_organ_flags`

Also, perhaps use `IS_ORGANIC_ORGAN` instead of `!IS_ROBOTIC_ORGAN`.

Reply:
Yes to all the above via https://github.com/tgstation/tgstation/pull/76766/commits/3222a128d4e957067f4b1f6281a3eb26c7eef315